14-01-date-input/14-1-EN-date-input.sbv

```diff
@@ -98,19 +98,19 @@ but not all of them. What happened?
 If time is not set, current time is used.
 
 0:01:48.840,0:01:54.500
-I have about 4 PM now so Git used it to get this date.
+I have about 4 PM now, so Git used it to get this date.
 
 0:01:54.900,0:01:58.880
 Lets have a look at some more examples to get a better understanding of these subtle moments.
 
 0:01:59.460,0:02:03.260
-Imagine its 30th of January, 2018
+Imagine it's 30th of January, 2018
 
 0:02:03.560,0:02:05.079
 12:30 AM
 
-0:02:05.080,0:02:08.860
-I will show the actual dates on the left and the way git perceives them on the right.
+0:02:05.080,0:02:08.860 
+I will show the actual dates on the left side, and the way Git interprets them on the right.
```

Comment:
human perceives, program interprets)

Reply:
Yes, if this is meant as a standalone tutorial we definitely need to give more details. But I feel like its not enough data to be a standalone tutorial anyway. So whoever is going to read this will see the video. I don't mind either way, just wanted to make sure.